### ISSUE TITLE
CLI: Both skip and silent use the same short flag

### ISSUE THREAD

**Issue by ashur** [JustinBeckwith/linkinator#69]
## Description

The CLI uses `-s` as the short flag for both `skip` and `silent`:

```
    --skip, -s
        List of urls in regexy form to not include in the check.
```

```
    --silent, -s
        Only output broken links
```

## Notes

Currently, this seems to favor `silent`, meaning `linkinator ./ -s www.googleapis.com` is not recognized as a valid argument.

## Personal Notes

This is a great tool, thanks for building it! 🙏

**Comment by XhmikosR** [comment]
This is indeed true @JustinBeckwith, and I also missed it in #58

**Comment by JustinBeckwith** [comment]
Fixed in #72
